deployments/docker/veraison

```diff
@@ -9,20 +9,24 @@ function status() {
     local prov=$(_get_container_state provisioning-service)
     local verif=$(_get_container_state verification-service)
     local manage=$(_get_container_state management-service)
+    local keycloak=$(_get_container_state keycloak-service)
```

Comment:
Presumably: This has an impact on the diagram listed here:?
https://github.com/veraison/services/tree/main/deployments/docker